src/megameklab/com/printing/InventoryEntry.java

```diff
@@ -0,0 +1,156 @@
+     * @return    The upper limit of the short range (ground units) or SRV (aero units)
+     */
+    String getShortField(int row);
+
+    /**
+     * @param row The row index within the entry. Should be < nRows()
+     * @return    The upper limit of the medium range (ground units) or MRV (aero units)
+     */
+    String getMediumField(int row);
+
+    /**
+     * @param row The row index within the entry. Should be < nRows()
+     * @return    The upper limit of the long range (ground units) or LRV (aero units)
+     */
+    String getLongField(int row);
+
+    /**
+     * @param row The row index within the entry. Should be < nRows()
+     * @return    The ERV (aero units)
+     */
+    String getExtremeField(int row);
+
+    /**
+     * @return Whether lines after the first line should be indented
+     */
+    boolean indentMultiline();
+
+    /**
+     * Computes any modification to the aerospace AV for linked Artemis, Apollo, or PPC capacitor
+     *
+     * @param weapon The weapon mount
+     * @return       The AV modification, if any
+     */
+    default int aeroAVMod(Mounted weapon) {
+        if (weapon.getType() instanceof WeaponType && weapon.getLinkedBy() != null) {
+            return aeroAVMod((WeaponType) weapon.getType(),
+                    weapon.getLinkedBy().getType(), weapon.getEntity().usesWeaponBays());
+        }
+        return 0;
+    }
+
+    /**
+     * Computes any modification to the aerospace AV for linked Artemis, Apollo, or PPC capacitor
+     *
+     * @param weapon   The type of weapon
+     * @param linkedBy The type of equipment the weapon is linked by
+     * @param bay      Whether the weapon is part of a weapon bay
+     * @return         The AV modification, if any
+     */
+    default int aeroAVMod(WeaponType weapon, EquipmentType linkedBy, boolean bay) {
+        if (linkedBy.hasFlag(MiscType.F_ARTEMIS)
+                || linkedBy.hasFlag(MiscType.F_ARTEMIS_V)) {
+            // The 9 and 10 rows of the cluster hits table is only different in the 3 column
+            if (weapon.getAtClass() == WeaponType.CLASS_MML) {
+                if (weapon.getRackSize() >= 7) {
+                    return 2;
+                } else if ((weapon.getRackSize() >= 5) || linkedBy.hasFlag(MiscType.F_ARTEMIS_V)) {
+                    return 1;
+                }
+            } else if (weapon instanceof LRMWeapon) {
+                return weapon.getRackSize() / 5;
+            } else if (weapon instanceof SRMWeapon) {
+                return 2;
+            }
+        } else if (linkedBy.hasFlag(MiscType.F_ARTEMIS_PROTO) && weapon.getRackSize() == 2) {
+            // The +1 only makes a difference for SRM2
+            return 2;
+        } else if (linkedBy.hasFlag(MiscType.F_APOLLO) && weapon.getRackSize() == 2) {
+            // The -1 only makes a difference for SRM2
+            return -2;
+        } else if (bay && linkedBy.hasFlag(MiscType.F_PPC_CAPACITOR)) {
```

Comment:
The comments mention +1 and -1, but they return +2 and -2. Is this intended? (I'm unfamiliar with this area of the code)

Reply:
Yeah, that's confusing. The +/- 1 is to the cluster hits table. The +/- 2 is to the damage because of one more missile hit on average. But that doesn't seem right because that would be a difference of two missIles between 6 and 8. I'll double check that when I get home.

Reply:
I think I was looking at the wrong pair of rows. The -1 cluster hit mod for Apollo doesn't affect SRM2, but it does reduce the AV for SRM4. I fixed that and made the comments actually helpful.

Reply:
So apparently I was doing this section on autopilot and didn't actually think about it until now. Apollo is for MRMs, which comes in rack sizes of 10, 20, 30, and 40. It has no affect on the aerospace AV. I fixed it properly this time.